internal/rbd/nodeserver.go

```diff
@@ -880,9 +880,9 @@ func (ns *NodeServer) mountVolume(ctx context.Context, stagingPath string, req *
 
 func (ns *NodeServer) createTargetMountPath(ctx context.Context, mountPath string, isBlock bool) (bool, error) {
```

Comment:
can we update the `createTargetMountPath` to return isMnt to check it simple, returning `!` and doing `!` check on the function return value looks so confusing :(

Reply:
Sure, I can clean that up. It confused me a little too.